src/urllib3/connection.py

```diff
@@ -4,7 +4,7 @@
 import re
 import socket
 import warnings
-from http.client import HTTPConnection as _HTTPConnection
```

Comment:
Can we revert the change here? Ensures that HTTPConnection name can only be one thing throughout this file.

Reply:
done.